GenderPayGap.WebUI/BusinessLogic/Services/UpdateFromCompaniesHouseService.cs

```diff
@@ -193,54 +193,6 @@ public static OrganisationAddress CreateOrganisationAddressFromCompaniesHouseAdd
             };
         }
 
-        private static string GetAddressLineFromPremisesAndAddressLine1(CompaniesHouseAddress companiesHouseAddress)
-        {
-            return companiesHouseAddress?.Premises == null
-                ? companiesHouseAddress?.AddressLine1
-                : companiesHouseAddress?.Premises + "," + companiesHouseAddress?.AddressLine1;
-        }
-
-        public static bool AddressMatches(OrganisationAddress firstOrganisationAddress, OrganisationAddress secondOrganisationAddress)

```

Comment:
This function exists on the `OrganisationAddress` class already! Refactored to use that, calling it with `firstOrganisationAddress.AddressMatches(secondOrgansiationAddress)` instead :)